pkg/controller/networkpolicy/clusternetworkpolicy_test.go

```diff
@@ -885,7 +911,74 @@ func TestProcessClusterNetworkPolicy(t *testing.T) {
 			expectedAppliedToGroups: 2,
 			expectedAddressGroups:   2,
 		},
+		{
+			name: "rule-with-to-service",
+			inputPolicy: &crdv1alpha1.ClusterNetworkPolicy{
+				ObjectMeta: metav1.ObjectMeta{Name: "cnpK", UID: "uidK"},
+				Spec: crdv1alpha1.ClusterNetworkPolicySpec{
+					AppliedTo: []crdv1alpha1.NetworkPolicyPeer{
+						{PodSelector: &selectorA},
+					},
+					Priority: p10,
+					Egress: []crdv1alpha1.Rule{
+						{
+							ToService: []*crdv1alpha1.ServiceReference{
+								{
+									Name:      "svcA",
```

Comment:
it seems that `svcA` doesn't actually exist, that doesn't impact the test at all?

Reply:
During the process, we get the Endpoints using ServiceName and ServiceNamespace and then use the Endpoints to create rules.